src/tiled/main.cpp

```diff
@@ -180,6 +218,49 @@ void CommandLineHandler::startNewInstance()
     newInstance = true;
 }
 
+template <typename T>
+inline T *CommandLineHandler::findExportFormat(const QString *filter,
+                                               const QString &targetFile)
+{
+    T *outputFormat = nullptr;
+    const auto formats = PluginManager::objects<T>();
+
+    if (filter) {
+        // Find the format supporting the given filter
+        for (T *format : formats) {
+            if (!format->hasCapabilities(T::Write))
+                continue;
+            if (format->shortName().compare(*filter, Qt::CaseInsensitive) == 0) {
+                outputFormat = format;
+                break;
+            }
+        }
+        if (!outputFormat) {
+            mError = "Format not recognized (see --export-formats)";
```

Comment:
You should make `mError` a `QString` and perform the translation here, otherwise `lupdate` will not be able to recognize that this is a translatable string.